File: EntertainmentPortal/EP.15Puzzle.Logic/Models/Deck.cs
using System.Collections.Generic;

namespace EP._15Puzzle.Logic.Models
{
    public class Deck
    {
        
        public int Score { get; set; }
        public bool Victory { get; set; }
        public int Size { get; set; }
        public Tile[,] Tiles { get; set; }

        
        
    }
}

File: EntertainmentPortal/ep-pyatnashki-front/src/app/pyatnashki/models/deck.ts
import { Tile } from './tile';
export class Deck {
  size: number;
  score: number;
  victory: boolean;
  tiles: Tile[];

  constructor(score: number, size: number, victory: boolean, tiles: Tile[]) {
    this.score = score;
    this.size = size;
    this.victory = victory;
    this.tiles = tiles;
  }
}
